app/queries/get_application_choices_for_providers.rb

```diff
@@ -4,7 +4,7 @@ class GetApplicationChoicesForProviders
     :offered_course_option,
     :provider,
     :site,
-    application_form: %i[candidate application_references application_work_experiences application_volunteering_experiences application_qualifications],
+    application_form: %i[candidate application_references application_work_experiences application_volunteering_experiences application_qualifications application_work_history_breaks],
```

Comment:
Should this go in a different commit, with an explanation for why we're adding it?

Reply:
Have done 👍